### PR TITLE
First draft CC Focal Use Case

### PR DESCRIPTION
**Pull request by GraceRachmany**
Joe made me do it.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -72,3 +72,79 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
...
+ * Most community currencies can’t be exchanged directly for a fiat currency as a means of exchange.
+ * Generally community currencies have closed software systems, without interfaces to other currencies or exchanges.
+ 
+ ### Artifacts
+ 1. Multi-coin wallet for community currencies.
+ 2. Interchain currency exchange standard
+ 3. IID for local currencies -- IID uniquely identifies local currencies, providing a moniker that can be applied without ambiguity across any exchange. In other words, there’s no confusion about which chain gets to be called “BTC” or similar market problems.
+    a. Circles UBI
+    b. Sarafu
+    c. Eusko
```
Comment (jandrieu) [inline index.md]:
```suggestion
```

File: index.md [hunk 2]
```diff
@@ -72,3 +72,78 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
...
+ * Most community currencies can’t be exchanged directly for a fiat currency as a means of exchange.
+ * Generally community currencies have closed software systems, without interfaces to other currencies or exchanges.
+ 
+ ### Artifacts
+ 1. Multi-coin wallet for community currencies.
+ 2. Interchain currency exchange standard
+ 3. IID for local currencies -- IID uniquely identifies local currencies, providing a moniker that can be applied without ambiguity across any exchange. In other words, there’s no confusion about which chain gets to be called “BTC” or similar market problems.
+    a. Circles UBI
+    b. Sarafu
+    d. SEEDS
```
Comment (jandrieu) [inline index.md]:
```suggestion
    c. SEEDS
```

File: index.md [hunk 3]
```diff
@@ -72,3 +72,78 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
...
+ 
+ ( I believe we agreed to split this into two use cases, keeping the purely crypto-crypto exchange for the focal use case and raising the crypto -> non-crypto use case in the domain map rather than as a focal use case.
```
Comment (jandrieu) [inline index.md]:
```suggestion
```

**Comment by jandrieu** [comment]
For issue #8

File: index.md [hunk 4]
```diff
@@ -72,3 +72,79 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
...
+## Hungry Lisbon Traveler
+Contributed by Grace Rachmany
+
+### Background
+ In the context of creating an alternative to the default economy, people want to be able to use and exchange alternative currencies, using their alternative currency wallets as easily as they use a credit card. When people use a credit card, neither the buyer nor the seller needs to worry about the exchange rate--the transaction happens seamlessly. The seller receives their local currency, and the buyer is debited in their local currency.
+ 
+ The vision of the ICF, and the cryptocurrency movement as a whole, is that people will be free to use whatever means of exchange they choose, and that communities will be free to create their own means of exchange.
+ 
+ ### Scenario
+ Grace is an advisor to different community currency groups. She has a Circles and a SEEDS wallet and some Sarafu. While in Basque country, she is looking for a restaurant that accepts community currencies. She uses the Open Street Maps to identify a restaurant which accepts community currency, in this case Chiemgauer. Her wallet allows her to convert Circles to Eusko and buy a meal for herself and a colleague at Nina’s Cafe.
```
Comment (wip-abramson) [inline index.md:87]: Are there two currencies referenced here? I.e. Chiemgauer and Eusko

File: index.md [hunk 5]
```diff
@@ -72,3 +72,79 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
...
+ ### Distinction / Challenge
+ * Different currencies have different issuance mechanisms (mutual credit, reserve backing, airdrop, UBI, backing based on goods).
+ * Most community currencies can’t be exchanged directly for a fiat currency as a means of exchange.
+ * Generally community currencies have closed software systems, without interfaces to other currencies or exchanges.
+ 
+ ### Artifacts
+ 1. Multi-coin wallet for community currencies.
+ 2. Interchain currency exchange standard
+ 3. IID for local currencies -- IID uniquely identifies local currencies, providing a moniker that can be applied without ambiguity across any exchange. In other words, there’s no confusion about which chain gets to be called “BTC” or similar market problems.
+    a. Circles UBI
```
Comment (wip-abramson) [inline index.md:100]: Eusko or Chiemgauer also?

File: index.md (hunk 4 above)
Comment (alwyn-ixo) [reply index.md:87]: @jandrieu Same for this PR; do you want to update based on  Will's comment?

**Review by alwyn-ixo — APPROVED** [review]

**Review by alwyn-ixo — APPROVED** [review]

**Review by alwyn-ixo — APPROVED** [review]
Forced the merge - could no longer wait for a review of comments.

**Pull request merged**